src/app/component/Input/Input.style.scss

```diff
@@ -0,0 +1,3 @@
+.Input {
+    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4724211)